New Dev Docs introduction (#1911)
strapi/documentation: docusaurus/docs/dev-docs/intro.md

# Welcome to the Strapi Developer Docs!

<SubtleCallout title="Developer Docs, User Guide, and Strapi Cloud documentation" emoji="📍">

The documentation for Strapi contains 3 main sections, accessible from the top navigation bar:

- 🧑‍🏫 The **[User Guide](/user-docs/intro)** is all about using Strapi's admin panel.
- ☁️ The **[Strapi Cloud](/cloud/intro)** documentation is about deploying your Strapi application to Strapi Cloud and managing your Strapi Cloud projects and settings.

> - 🧑‍💻 The **Developer Docs** that you're currently reading contain all the technical information related to the setup, advanced usage, customization, and update of your Strapi v5 application.

Reviewer: I would suggest using the same order as in the top navigation bar, to make it even more coherent 😊

</SubtleCallout>

> 👉 If you're totally new to **Strapi** <Annotation>**💡 Did you know?**<br />The original purpose of the project was to help Boot**strap** your **API**: that's where the Strapi name comes from and how Strapi was created.<br /><br />Now, Strapi is an **open-source headless CMS** that gives developers the freedom to choose their favorite tools and frameworks and allows editors to manage and distribute their content using their application's admin panel.<br /><br />Based on a plugin system, Strapi is a flexible CMS whose admin panel and API are extensible — and which every part is customizable to match any use case. Strapi also has a built-in user system to manage in detail what the administrators and end users have access to.<br /></Annotation> or if you're a developer who'd like to take the most out of Strapi, the Developer Docs are the place to start! We suggest you to:

Reviewer: ```suggestion
👉 If you're totally new to **Strapi** <Annotation>**💡 Did you know?**<br />The original purpose of the project was to help Boot**strap** your **API**: that's where the Strapi name comes from and how Strapi was created.<br /><br />Now, Strapi is an **open-source headless CMS** that gives developers the freedom to choose their favorite tools and frameworks and allows editors to manage and distribute their content using their application's admin panel.<br /><br />Based on a plugin system, Strapi is a flexible CMS whose admin panel and API are extensible — and which every part is customizable to match any use case. Strapi also has a built-in user system to manage in detail what the administrators and end users have access to.<br /></Annotation> or if you're a developer who would like to take the most out of Strapi, the Developer Docs are the place to start! We suggest you to:
```

1. Head over to the [Quick Start](/dev-docs/quick-start) guide.

> 2. Continue your journey through the various Developer Docs sections. The ["What you will learn"](#what-you-will-learn) section gives you an overview of the available topics.

Reviewer: I've noticed that here you use the word "sections" but a bit below, in "What you will learn", you use the word "categories". I think it would be clearer to choose one word and stick to it 🤓

> ## What you will learn

Reviewer: Completely open question: should we name that title "What you will learn" or something like "What you can learn about"/"What you will find here"? Basically, should we choose a title that emphasise that the user's journey can be based on his choices and specific needs? I hope that makes sense.

Author: That's a good suggestion, thanks! I like the "What you will find here" even more because I feel like it emphasizes the journey ("here" vs. what is not here, if that makes sense 😄). I've updated it.

:::prerequisites
Strapi Developer Docs are mostly intended to developers and assumes some prior knowledge of the JavaScript ecosystem.

> If, while discovering Strapi, you also make your first steps with JavaScript web development, we encourage you to read more about [JavaScript](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics) and [npm](https://docs.npmjs.com/about-npm), and, if applicable to your project, [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html), before diving deeper into the Strapi Developer Docs.

Reviewer: ```suggestion
If you also make your first steps with JavaScript web development while discovering Strapi, we encourage you to read more about [JavaScript](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics) and [npm](https://docs.npmjs.com/about-npm). If applicable to your project, you can also learn about [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html) before diving deeper into the Strapi Developer Docs.
```
Little suggestion/reminder to keep sentences short, simple and to the point 😄

:::tip Tips
- If you already know exactly what you're searching for, use the search bar at the top or navigate using the table of content.
- If you prefer learning more about Strapi while looking at the project code structure, you can use the interactive [project structure](/dev-docs/project-structure) page.

> - If demos are more your thing, we have a [video demo](https://youtu.be/zd0_S_FPzKg), or you can request a [live demo](https://strapi.io/demo).

Reviewer: ```suggestion
- If demos are more your thing, feel free to watch the [video demo](https://youtu.be/zd0_S_FPzKg), or you can request a [live demo](https://strapi.io/demo).
```
I don't know if it's still a thing but at some point we were encouraged to not use "we" too much so just in case it's still relevant, I suggest this little modification!
